python/galois/cpp/libgalois/graphs/Util.pxd

```diff
@@ -0,0 +1,12 @@
+# Omit the exception specifications here to
+# allow returning lvalues.
+# Since the exception specifications are omitted here,
+# these classes/functions ABSOLUTELY MUST be used only
+# within functions with C++ exception handling specifications.
+# This is intentional and is required to ensure that C++ exceptions
+# thrown in the code written using these forward declarations
+# are forwarded properly into the Galois library rather than
+# being converted into Python exceptions.
```

Comment:
It appears that this comment no longer applies to anything in this file. Related to it though: with https://github.com/cython/cython/pull/3362 merged and hopefully released soon, we could stop worrying about this with non-copyable types provided that we're returning movable objects.